client/src/main/com/sinch/sdk/domains/voice/adapters/converters/SAVMLActionDtoConverter.java

```diff
@@ -240,10 +244,19 @@ private static List<OptionDto> convertMenuOptionCollection(Collection<MenuOption
     }
     return client.stream()
         .map(
-            f ->
-                new OptionDto()
-                    .action(EnumDynamicConverter.convert(f.getAction()))
-                    .dtmf(DualToneMultiFrequencyDtoConverter.convert(f.getDtfm())))
+            f -> {
+              OptionDto dto = new OptionDto();
+              f.getAction().ifPresent(f2 -> dto.action(convert(f2)));
+              f.getDtfm().ifPresent(f2 -> dto.dtmf(DualToneMultiFrequencyDtoConverter.convert(f2)));
+              return dto;
+            })
         .collect(Collectors.toList());
   }
+
+  private static String convert(MenuOptionAction client) {
+    if (null == client) {
+      return null;
+    }
+    return String.format("%s(%s)", client.getType().value(), client.getId());
```

Comment:
There will be a NPE if the object is built with `MenuOptionAction.from(null, "foo")`

Reply:
Fixed, added: 
- a guard against null pointer onto format
- a validation against null value onto menu constructor because of it does have not a lot of sense to set action to null